libmscore/cmd.cpp

```diff
@@ -268,7 +268,7 @@ void Score::endCmd(bool rollback)
             masterScore()->setPlaylistDirty();  // TODO: flag individual operations
             masterScore()->setAutosaveDirty(true);
             }
-      MuseScoreCore::mscoreCore->endCmd();
+      //MuseScoreCore::mscoreCore->endCmd();
```

Comment:
I would rather avoid an extra call in `MuseScore::cmd()` than disabling this call. `Score::endCmd()` does not always result from `MuseScore::cmd()` call, and `MuseScore::endCmd()` should currently always be called after each score change. Disabling this line may happen to influence Inspector tests as well, as Inspector code calls exactly `Score::endCmd()`, not `MuseScore::endCmd()`.

Reply:
Yes I'm starting to suspect that's the issue with the test, too.  I knew this change would need further testing, looks like the script test may have done it for me!

FWIW, calling MuseScore::endCmd() twice isn't problematic in itself, if we could guarantee only one call to updateAccessibility().

I'll look into making the change in MuseScore::cmd().  FWIW, the reason calling updateAccessibility twice is a problem is that it causes us to send two QAccessibleEvents.  NVDA omehow is OK with that, but other screenreaders will try to read twice, once I managed to get them to read at all.